modules/common-workflows/pages/workflow-sp-upgrade-webui.adoc

```diff
@@ -0,0 +1,61 @@
+[[workflow-sp-upgrade-via-webui]]
+= Service Pack Upgrade via {webui}
+
+
+
+If you want to migrate the registered {sle} client's service pack to a newer version, it can be done either on the command line or via {webui}.
+
+This document describes and illustrates in detail the migration using the {webui}.
+
+[NOTE]
+====
+SP versions used are not reflective of the alctual latest versions available.
+They are used for illustration purposes only.
+====
+
+
+.Procedure: Migrating service pack to a newer version using Web UI
+[role=procedure]
+
+
+. Login to SUMA Web UI and Go to menu:Admin[Products] and search for "SUSE Linux Enterprise Server 15 SP4 x86_64 (BETA)"
+. Select the recommended channels as shown.
++
+image::upgradesles01.png[scaledwidth=80%]
+. Click btn:[Add Products] as shown below
++
+image::upgradesles02.png[scaledwidth=80%]
+. Go to menu:Systems[registered client > Software > SP Migration]. It will show you 2 targets, SP3 and SP4. 
++
+image::upgradesles03.png[scaledwidth=80%]
+. Select literal:[SUSE Linux Enterprise Server 15 SP4 x86_64 (BETA)]. This will expand further as shown below.
++
+image::upgradesles04.png[scaledwidth=80%]
+. Select [literal]``Target Base Channel`` as [literal]``SLE-Product-SLES15-SP4-Pool for x86_64`` and keep [literal]``Allow Vendor Change`` unchecked.
+. Click on btn:[Schedule Migration] and it will highlight message [literal]``It is better to do a dry run first so continuing with dry run first.``
+. Click on btn:[Dry-run] and check the status of the simulation in menu:Events[History]. You should see a return code [literal]``0`` meaning successful.
+. Now you may click on btn:[Schedule Migration] to actually migrate the server. Following message will get highlighted on top of the screen in SUMA Web UI.
```

Comment:
```suggestion
. Click btn:[Schedule Migration] and it will highlight message [literal]``It is better to do a dry run first so continuing with dry run first.``
. Click btn:[Dry-run] and check the status of the simulation in menu:Events[History].
  You should see a return code [literal]``0`` meaning successful.
. Now you may click btn:[Schedule Migration] to actually migrate the server.
  Following message will get highlighted on top of the screen in {productname} {webui}.
```